PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -421,6 +411,18 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
         dEsigmaDown=Var("userFloat('ecalTrkEnergyPostCorrNew')-userFloat('energySigmaDownNew')", float,  doc="ecal energy smearing value shifted 1 sigma up", precision=8),
 
 )
+##Keeping the possibilty of using V1 working points in older eras
+(~run2_nanoAOD_106Xv2).toModify(electronTable.variables,
```

Comment:
This will not work, as it add back in the past and future eras.
So I suggest to keep in the ones before the UL

`(run2_nanoAOD_92X | run2_nanoAOD_94XMiniAODv1 | run2_nanoAOD_94XMiniAODv2 | run2_nanoAOD_94X2016 | run2_nanoAOD_102Xv1).toModify( .... ) `

same in the 
https://github.com/cms-sw/cmssw/pull/33701/files#diff-5f4338f7d7671c8348d9097ff44f255cc0b36d204c639782856a7c36cbc8fbdaR75